@xen-orchestra/lite/src/stories/web-core/ui/input/ui-input.story.vue

```diff
@@ -1,19 +1,33 @@
 <template>
   <ComponentStory
-    v-slot="{ properties }"
+    v-slot="{ properties, settings }"
     :params="[
       model().type('string').preset(''),
```

Comment:
To match the `modelValue` typing in the component:
```suggestion
      model().type(`string | number`).preset(''),
```